pkg/controller/migmigration/task.go

```diff
@@ -467,6 +468,11 @@ func (t *Task) UID() string {
 	return string(t.Owner.UID)
 }
 
+// Migration Stage Pod Search Label
+func (t *Task) StagePodSearchLabel() string {
+	return fmt.Sprintf("%s-search", t.UID())
```

Comment:
Related to the other comment, I'm not sure a (1) line function that is used in (2) places is worthwhile.

Reply:
Suggest adding something like: `The -excluded suffix not matched by the label selector.` to the label comment.

https://github.com/fusor/mig-controller/blob/b1e56b34baac15a269b2b2b4511605a6f11db34b/pkg/controller/migmigration/annotation.go#L31